src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawnable, AreaCalculator {
+    private String figureColor;
+
+    public Figure(String figureColor) {
+        this.figureColor = figureColor;
+    }
+
+    public String getFigureColor() {
+        return figureColor;
+    }
+
+    public void setFigureColor(String figureColor) {
+        this.figureColor = figureColor;
+    }
+
+    public abstract double obtainArea();
+
+    public abstract void drawFigure();
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-abstract-classes-to-set-behaviour-for-classes)